wallet.go

```diff
@@ -147,3 +148,91 @@ func (w *Wallet) SendTransaction(tx *types.Transaction) (id string, err error) {
 	}
 	return
 }
+
+// about bundle tx
+
+func (w *Wallet) CreateBundleDataItem(data []byte, signatureType int, target string, anchor string, tags []types.Tag) (di DataItem, err error) {
```

Comment:
Should be split
bundles := CreateBundleDateItem()
e := w.SignBundle(bundles)

Reference: utils.SignTransaction